webcompat/static/js/lib/comments.js

```diff
@@ -7,7 +7,16 @@ var issues = issues || {};
 issues.CommentsCollection = Backbone.Collection.extend({
   model: issues.Comment,
   url: function() {
-    return '/api/issues/' + issueNumber + '/comments?per_page=100';
+    return '/api/issues/' + issueNumber + '/comments?page=' + this.defaultPage;
+  },
+  initialize: function(param) {
+    this.defaultPage = param[0].page;
+  },
+  fetchPage: function(Commentpage, options) {
```

Comment:
The case for `Commentpage` here is probably better as `commentPage` (typically in JS something starting with a capital should be a constructor function).